docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,370 @@
+    "\n",
+    "Note that to execute it (in a command line environment), you'll need to remove the ! from the beginning of the lines.\n",
+    "\n",
+    "**Getting help on the script**: In the following cell, we show how to get help on the script."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "usage: atomdb [-h] (-c | -q) [-e EXC] dataset elem charge mult\n",
+      "\n",
+      "Compile or query an AtomDB entry.\n",
+      "\n",
+      "options:\n",
+      "  -h, --help         show this help message and exit\n",
+      "\n",
+      "commands:\n",
+      "  -c, --compile      compile a species into the database\n",
+      "  -q, --query        query a species from the database\n",
+      "\n",
+      "arguments:\n",
+      "  dataset            name of dataset\n",
+      "  elem               element symbol\n",
+      "  charge             charge\n",
+      "  mult               multiplicity\n",
+      "  -e EXC, --exc EXC  excitation level\n"
+     ]
+    }
+   ],
+   "source": [
+    "!python -m atomdb -h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Printing atom properties using the script:**\n",
+    "\n",
+    "The data is printed as human readable following the JSON format. This can be latter parsed and used \n",
+    "in other scripts or programs. The following cell shows how to print the data for the Carbon atom with\n",
+    "charge 0 and spin multiplicity 3 (ground state). Due to the large amount of data, the output stored\n",
+    "in the variable c_0_3_json_output sn only three properties are parsed and printed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "elem: C\n",
+      "atnum: 6\n",
+      "atmass: 21893.98446012775\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash \n",
+    "c_0_3_json_output=$(python -m atomdb -q slater C 0 3)\n",
+    "\n",
+    "# Define an array of keys\n",
+    "keys=(\"elem\" \"atnum\" \"atmass\")\n",
+    "\n",
+    "# Loop through the keys and print the raw-values from the JSON output\n",
+    "for key in \"${keys[@]}\"; do\n",
+    "    value=$(echo \"$c_0_3_json_output\" | jq -r \".$key\")\n",
+    "    echo \"$key: $value\"\n",
+    "done"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use as a Python library\n",
+    "\n",
+    "The real power of AtomDB is showcased when it is used as a library. The library provides a simple API to access atomic properties and even some pro-molecular properties this is done through the `Species` object, which is used to access atomic properties. \n",
```

Comment:
Another minor issue:
``...some pro-molecular properties. This is done..."

Reply:
Done! Thanks!!